cpmonitor/models.py

```diff
@@ -136,6 +134,65 @@ def validate_unique(self, exclude=None):
             raise ValidationError(msgs)
 
 
+class ChecklistClimateActionPlan(models.Model):
+    city = models.OneToOneField(
+        City,
+        models.PROTECT,
+        name="KAP Checkliste",
+        related_name="checklist_climate_action_plan",
+    )
+
+    climate_action_plan_exists = models.BooleanField(
+        "Gibt es einen KAP?", default=False
```

Comment:
I use this text now in the admin and the frontend view. Are the question understandable for the general public especially some abbreviations like KAP, LULUCF, THG-Emissions etc.? Or do we need to rephrase them?

Reply:
Ok, I created an issue #171